DiceKitTests/Die_Tests.swift

```diff
@@ -404,4 +404,38 @@ extension Die_Tests {
         expect(result.value) == stubbedRoll
     }
     
+    func test_probabilityMass_shouldReturnCorrect() {
+        property("probability mass") <- forAll {
+            (sides: Int) in
+            
+            let inverseSides = -sides
+            
+            let die = Die(sides: sides)
+            let inverseDie = Die(sides: inverseSides)
+            
+            let probMass = die.probabilityMass
+            let inverseProbMass = inverseDie.probabilityMass
+            
+            guard sides != 0 else {
```

Comment:
Also, spacing is fine. You can have more logical groups that the minimum. Don't be afraid to add space.